docs/guides/installing-the-slack-cli-for-mac-and-linux.md

```diff
@@ -165,8 +165,7 @@ In addition, if you attempt to run the `slack run` command without this dependen
 
 Ensure that `deno-slack-hooks` is installed at the project level and that the version is not less than `v1.3.0`.
 
-**8\. [Install the VSCode extension for
-   Deno](/deno-slack-sdk/guides/installing-deno#vscode) (recommended).**
+**8\. [Install the VSCode extension for Deno](/deno-slack-sdk/guides/installing-deno#vscode) (recommended).**
```

Comment:
question: Soon, it may make sense to have the Deno install instructions on the Deno SDK page and link to it (and Bolt JS/Python) from here.